README.md

```diff
@@ -1,27 +1,20 @@
+[![Build Status](https://travis-ci.org/FourthState/plasma-mvp-sidechain.svg?branch=develop)](https://travis-ci.org/FourthState/plasma-mvp-sidechain)
+[![codecov](https://codecov.io/gh/FourthState/plasma-mvp-sidechain/branch/develop/graph/badge.svg)](https://codecov.io/gh/FourthState/plasma-mvp-sidechain)
+[![Discord](https://img.shields.io/badge/discord-join%20chat-blue.svg)](https://discord.gg/YTB5A4P)
 [![license](https://img.shields.io/github/license/FourthState/plasma-mvp-rootchain.svg)](https://github.com/FourthState/plasma-mvp-sidechain/blob/master/LICENSE)
 
-Branch    | Tests | Coverage
-----------|-------|----------
-develop   | [![Build Status](https://travis-ci.org/FourthState/plasma-mvp-sidechain.svg?branch=develop)](https://travis-ci.org/FourthState/plasma-mvp-sidechain) | [![codecov](https://codecov.io/gh/FourthState/plasma-mvp-sidechain/branch/develop/graph/badge.svg)](https://codecov.io/gh/FourthState/plasma-mvp-sidechain)
-master	  | [![Build Status](https://travis-ci.org/FourthState/plasma-mvp-sidechain.svg?branch=master)](https://travis-ci.org/FourthState/plasma-mvp-sidechain) | [![codecov](https://codecov.io/gh/FourthState/plasma-mvp-sidechain/branch/master/graph/badge.svg)](https://codecov.io/gh/FourthState/plasma-mvp-sidechain)
-
-This is the latest [Minimum Viable Plasma](https://ethresear.ch/t/minimal-viable-plasma/426) version.  
-
-**Note**: This sidechain is being constructed to be compatible with our [rootchain contract](https://github.com/FourthState/plasma-mvp-rootchain)  
+Implementation of [Minimum Viable Plasma](https://ethresear.ch/t/minimal-viable-plasma/426) compatible with our [rootchain contract](https://github.com/FourthState/plasma-mvp-rootchain)  
 
 ## What is Plasma?
 Plasma has two major components: verification and computation. 
-Verification is handled by the rootchain smart contract, which resolves any disputes and distributes funds accordingly. 
+Verification is handled by the rootchain contract, which resolves any disputes and distributes funds accordingly.
+ Computation is handled separately by a sidechain, which maintains its security through reporting proofs via merkle roots to the rootchain contract. 
 
-Computation is handled separately by a sidechain, which leverages the Cosmos SDK to create a modular and flexible blockchain.
-This sidechain maintains its security through reporting proofs via merkle roots to the Ethereum mainchain. 
-We will be using [Tendermint](https://github.com/tendermint/tendermint) as a consensus algorithm.
+Plasma MVP utilizes a UTXO model, which allows for secure and compact proofs. Learn more about plasma on [learnplasma.org](https://www.learnplasma.org/en/)!
 
-Plasma MVP utilizes a UTXO model, which allows for secure and compact proofs when interacting with the rootchain smart contract. 
-
-Learn more about plasma on [learnplasma.org](https://www.learnplasma.org/en/)!
+We are using [Tendermint](https://github.com/tendermint/tendermint) for our consensus protocol.
```

Comment:
Maybe a note that it currently supports only single validator but plans are in the works to extend to multiple validator?